Fix invalid long_description ruby code

szymonpk opened the pull request:
It is invalid and it wouldn't work. New line can be placed after `(`, not before.

mmzyk commented:
Note that if https://github.com/chef/chef-server/pull/465 is merged then these docs will need to revert back.

mmzyk commented:
This is no longer needed, closing.

Commit:
Fix invalid long_description ruby code

It is invalid and it wouldn't work. New line can be placed after `(`, not before.

Commit:
Replicated shared content changes

Signed-off-by: David Wrede <dwrede@chef.io>

davidwrede commented:
![gif-keyboard-5158513278730573385](https://cloud.githubusercontent.com/assets/1634752/25204627/4720fac2-2513-11e7-87f1-ae1ca5604fd2.gif)
